obs-service-tar_scm.spec

```diff
@@ -0,0 +1,166 @@
+#
+# spec file for package obs-service-tar_scm
+#
+# Copyright (c) 2017 SUSE LINUX GmbH, Nuernberg, Germany.
+#
+# All modifications and additions to the file contributed by third parties
+# remain the property of their copyright owners, unless otherwise agreed
+# upon. The license for this file, and modifications and additions to the
+# file, is the same license as for the pristine package itself (unless the
+# license for the pristine package is not an Open Source License, in which
+# case the license is the MIT License). An "Open Source License" is a
+# license that conforms to the Open Source Definition (Version 1.9)
+# published by the Open Source Initiative.
+
+# Please submit bugfixes or comments via http://bugs.opensuse.org/
+#
+
+
+%bcond_without obs_scm_testsuite
+
+Name:           obs-service-tar_scm
+Version:        0.7.0
+Release:        0
+Summary:        An OBS source service: checkout or update a tar ball from svn/git/hg
+License:        GPL-2.0+
+Group:          Development/Tools/Building
+Url:            https://github.com/openSUSE/obs-service-tar_scm
+Source:         %{name}-%{version}.tar.gz
+BuildRequires:  bzr
+BuildRequires:  git-core
+BuildRequires:  mercurial
+BuildRequires:  python >= 2.6
+%if 0%{?fedora_version} || 0%{?rhel_version} || 0%{?centos_version}
+BuildRequires:  PyYAML
+%else
+BuildRequires:  python-PyYAML
+%endif
+BuildRequires:  python-dateutil
+BuildRequires:  python-lxml
+BuildRequires:  python-mock
```

Comment:
I doubt, because it is not in the spec file of [openSUSE:Tools](https://build.opensuse.org/package/view_file/openSUSE:Tools/obs-service-tar_scm/obs-service-tar_scm.spec?expand=1) but this package  builds.

Please merge and lets fix problems afterwards